deploy/clowdapp.yaml

```diff
@@ -118,3 +120,7 @@ parameters:
   displayName: Sources Service Port
   description: Port to use for the Sources service URL.
   value: "8000"
+- name: AWS_WAIT_TIME
+  displayName: AWS Wait Time
+  description: Time to sleep between creating the resources and posting back to Sources API
+  value: "15"
```

Comment:
Is this the default value you set for clowder or is it just a placeholder? Because if it's a default value, maybe it should match the `const DEFAULT_SLEEP_TIME = 7`

Reply:
Correct, this is just the default for clowder. The reason I have it set higher here is because we want it higher in stage/prod without having to override it in `deploy-clowder.yaml`, but we have the power to increase/decrease it there if we want.

For running it locally (or outside of clowder) I just left the old default basically rather than requiring the env var to be set.

Reply:
All right, makes a lot of sense. Thanks!